### ISSUE TITLE
can NaNs be inferred in a network

### ISSUE THREAD

**Issue by claudius-kienle** [claudius-kienle/self-supervised-depth-denoising#12]
Is it possible to infer nans in a neural network?
If so, the NaNs must not be mapped to a interger value (like 0) and the nan-mask could be removed.

In addition, the loss function must be adapted, as it would output NaN, if the prediction/label contains a nan value
